airbyte-integrations/connectors/source-zendesk-support/metadata.yaml

```diff
@@ -11,7 +11,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 79c1aa37-dae3-42ae-b333-d1c105477715
-  dockerImageTag: 2.2.1
```

Comment:
this should be treated as `patch` changes

Reply:
Fixed